.github/workflows/purge-cache.yml

```diff
@@ -0,0 +1,13 @@
+name: Purge Cache
+
+on:
+  workflow_call:
+
+jobs:
+  purge-jsdelivr-cache:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: gacts/purge-jsdelivr-cache@v1
```

Comment:
I checked the source code of this action and noticed that it makes separate requests for every URL purge. But there is also a POST endpoint to purge multiple caches by a single request. Considering purging caches for many files, I think we should use batch endpoint.

> curl -X POST \
  [http://purge.jsdelivr.net/](https://clicktime.symantec.com/15t5ZrVW89yGfvgAzx8Mu?h=5qY1Lcx4f2-Dseq_AtTbDXbpznpzdQwFTsIPKUdk3TI=&u=http://purge.jsdelivr.net/) \
  -H 'cache-control: no-cache' \
  -H 'content-type: application/json' \
  -d '{
"path": [
"/npm/test",
"/npm/test2",
"/npm/test3"
]
}'

PS: It works with up to 20 paths.

These are the informations that I got from jsDelivr by e-mail.

Reply:
🤔 thanks for the comment. then we need to create an action to use in our repository to be able to purge and have a more trustworthy solution.